airbyte-integrations/connectors/source-googleanalytics-singer/requirements.standard-test.txt

```diff
@@ -0,0 +1,2 @@
+-e ../../bases/base-python-test
```

Comment:
is this file used for anything?

Reply:
it's helpful for local development on standard tests